loopd/swapclient_server.go

```diff
@@ -110,9 +110,10 @@ func (s *swapClientServer) LoopOut(ctx context.Context,
 	}
 
 	return &looprpc.SwapResponse{
-		Id:          hash.String(),
-		IdBytes:     hash[:],
-		HtlcAddress: htlc.String(),
+		Id:               hash.String(),
+		IdBytes:          hash[:],
+		HtlcAddress:      htlc.String(),
+		HtlcAddressP2Wsh: htlc.String(),
```

Comment:
Doesn't this now return a `P2SH` address in the `HtlcAddressP2Wsh` field for old swaps?

Reply:
That is okay and intentional. The `HtlcAddress` is deprecated.